float64 initial_radius
---
bool result
